### DISCUSSION TITLE
Sales logic: How to create a quote/offer for a lead

### DISCUSSION THREAD

**Discussion by blu-IT** [YetiForceCompany/YetiForceCRM#17305] [How to]
Hello,

I noticed that quotes/offers _can only be created when a client is already a custumer listed in the Organizations module_.
But in my logic I have a lead / someone who is interested in my service & products and wants a offer - he is still _listed in the Leads module_ - not in the Organizations module. When he accepts the offer, _then he gets a real client_ and should be converted in the Organizations module.

So is there a way to create offers/quotes an select data from the Leads module instead of Organizations?

Thanks!

**Comment by Jianbin719** [comment]
Solution 1: 
Add another field instead of Organizations to fill in with lead

**Comment by Jianbin719** [comment]
Solution 2:
Add one more record in database vtiger_fieldmodulerel

Here is the code, I used to add "Campaign" to Scalculation moduel.
INSERT INTO vtiger_fieldmodulerel (fieldid, module, relmodule, status, sequence) VALUES ('1806', 'SCalculations', 'Campaigns');
![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/75652286/a4540578-5c5d-4098-bf2a-50092b167e8f)

**Comment by Jianbin719** [comment]
![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/75652286/4a46aacf-98c6-41a3-88a4-c0a710e0c040)

**Comment by blu-IT** [comment]
OK, thanks! I will test an report!
